### PR TITLE
Prefix package name for generated pojos 

### PR DESCRIPTION
**Pull request by johnksv**
As it could cause problems when pojos from same xsds were on class path

Prefix all package names of generated pojos with `no.digipost`

### CODE REVIEW

**Review by post-svejk — APPROVED** [review]
Ser greit ut. Bør la @eivinhb og/eller @velfundert ta en kikk, og ta stilling til om denne endringen er ok i offentlig repo og som Maven-artefakter. Alternativet er vel å holde det på en privat fork, som kun deployer til vårt interne Maven-repo.

**Comment by eivinhb** [comment]
It should be easy to test if this has backwards compatibility issues with the client and/or meldingsformidler. If that seems ok, then I see no issues here.

**Comment by johnksv** [comment]
Must bump a major version due to change in package names. Thus clients and MF must change imports to be compatible with the changes introduced in this PR.

File: pom.xml [hunk 1]
```diff
@@ -23,11 +23,13 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```
Comment (runeflobakk) [inline pom.xml:26]:
```suggestion
```
😉

**Review by runeflobakk — APPROVED** [review]
Røddig! 🏆 🥇

**Pull request merged**